app/models/project.rb

```diff
@@ -0,0 +1,3 @@
+class Project < ApplicationRecord
+validates_presence_of :title, :description, :status
```

Comment:
it is always preferred to write single attribute validation over multiple attributes validation in a single line, please refer [here](https://github.com/rubocop-hq/rails-style-guide#single-attribute-validations)